Detection3D[] detections
string filename
Image_Detections image_detections
